demo/list-items/myApp.js

```diff
@@ -6,8 +6,43 @@
   ])
 
   .config([
-    function() {
-      // nothing to do here, ATM
+    'extensionRegistryUtils',
+    'extensionRegistryProvider',
+    function(utils, extensionRegistryProvider) {
+      // example custom comparator, which was the current comparator prototype
+      // similar to the built in, but more granular control sets invalid values to -9999.
+      var isString = utils.isString,
+          isNaN = utils.isNaN,
+          isNumber = utils.isNumber,
+          isUndefined = utils.isUndefined,
+          invalidVal = -9999,
+          strToUsableNum = function(val) {
+            var test = Number(val);
+            return isNaN(test) ?
+                    invalidVal :
+                    test;
+          },
+          toWeight = function(value) {
+            return isNaN(value) ?
+                    invalidVal :
+                    isNumber(value) ?
+                      value :
+                      isString(value) ?
+                        strToUsableNum(value) :
+                        isUndefined(value) ?
+                          -1 :
+                          invalidVal;
+          };
+      // can optionally override the comparator with a custom
+      // comparator function
+      extensionRegistryProvider.setComparator(function(a, b) {
```

Comment:
Anticipated the potential for feature request while I'm working on it since its fairly trivial change, but I'm good with skipping.  Will eliminate & update.